start-from-zero/plus.js

```diff
@@ -0,0 +1,11 @@
+
+var plus = document.querySelector('.plus-sign');
+
+plus.addEventListener('click',function(){
+    if (plus.textContent = '+'){
```

Comment:
I won't go into too much detail now, but what do you think would happen when this particular callback gets invoked? I have a hunch I disagree. :P